src/main/kotlin/com/cjbooms/fabrikt/generators/client/ClientGeneratorUtils.kt

```diff
@@ -100,8 +111,33 @@ object ClientGeneratorUtils {
                     builder.addAnnotation(annotationSpec)
                 }
             }
+            if (it is BodyParameter) {
+                annotateBodyParameterWith?.invoke(it)?.let { annotationSpec ->
+                    builder.addAnnotation(annotationSpec)
+                }
+            }
             builder.build()
         }
         return this.addParameters(specs)
     }
+
+    /**
+     * Adds suspend as modified to the func spec so that i can be used with CoroutineFeign
+     */
```

Comment:
is this kdoc still valid? Is ti only for CoroutineFeign still. Remove doc entirely if not

Reply:
fixed e105d48